src/pelz/main.c

```diff
@@ -188,7 +195,14 @@ int main(int argc, char **argv)
           memcpy(path_id, argv[arg_index + 3], path_id_size);
         }
         pelz_log(LOG_DEBUG, "<path> set: %.*s", (int) path_id_size, path_id);
-        msg = (char *) calloc((12 + path_id_size), sizeof(char));
+        if (file_check(path_id))
+	{
+          printf("Invalid <path> provided (file inassesible)\n");
```

Comment:
Saying a file is inaccessible implies a permissions issue, but the file_check can fail if the file does not exist. I suggest

    printf("File %s is invalid.", path_id);

Reply:
Print changed